src/main/java/seedu/duke/Duke.java

```diff
@@ -0,0 +1,58 @@
+package seedu.duke;
+
+import java.util.Scanner;
+
+import seedu.duke.commands.Command;
+import seedu.duke.commands.Parser;
+import seedu.duke.commands.Ui;
+import seedu.duke.storage.Storage;
+import seedu.duke.storage.TaskList;
+import seedu.duke.tasks.Task;
+
+import java.util.ArrayList;
+
+public class Duke {
+    private final TaskList taskList;
+    private final Storage storage;
+
+    /**
+     * Class Constructor.
```

Comment:
Perhaps this JavaDoc should be changed to follow convention?

From the [coding standard](https://se-education.org/guides/conventions/java/intermediate.html#comments):
> In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... (not Return or Returning etc.)

Also in line with Java's built-in class conventions: https://github.com/nus-cs2103-AY2122S1/forum/issues/82